core/schema/src/main/liquibase/19.1.0/changelog.xml

```diff
@@ -142,4 +142,331 @@
       delete from monitoringlocations where id='';
     </sql>
   </changeSet>
+
+  <!-- NMS-6584: Upgrade to Quartz 2 schema -->
+  <!-- http://www.quartz-scheduler.org/documentation/quartz-2.x/migration-guide.html -->
+  <changeSet author="seth" id="19.1.0-upgrade-to-quartz-2-schema">
+
+    <!-- Drop tables that are no longer used -->
+
+    <dropTable tableName="qrtz_job_listeners"/>
+    <dropTable tableName="qrtz_trigger_listeners"/>
+
+
+    <!-- Drop columns that are no longer used -->
+
+    <dropColumn columnName="is_volatile" tableName="qrtz_fired_triggers"/>
+    <dropColumn columnName="is_volatile" tableName="qrtz_job_details"/>
+    <dropColumn columnName="is_volatile" tableName="qrtz_triggers"/>
+
+
+    <!-- SL 2017-01-16: Fix some additional inconsistencies in the Quartz schema -->
+
+    <dropColumn columnName="recoverer" tableName="qrtz_scheduler_state"/>
+    <dropNotNullConstraint columnDataType="bytea" columnName="job_data" tableName="qrtz_job_details"/>
+    <addNotNullConstraint columnName="priority" tableName="qrtz_fired_triggers"/>
+
+    <addColumn tableName="qrtz_fired_triggers">
+      <column name="sched_time" type="bigint"/>
+    </addColumn>
+    <sql>UPDATE qrtz_fired_triggers SET sched_time = fired_time;</sql>
+    <addNotNullConstraint columnName="sched_time" tableName="qrtz_fired_triggers"/>
+
+
+    <!-- Add new columns that replace the 'is_stateful' column -->
+
+    <addColumn tableName="qrtz_job_details">
+      <column name="is_nonconcurrent" type="boolean"/>
+    </addColumn>
+    <sql>UPDATE qrtz_job_details SET is_nonconcurrent = is_stateful;</sql>
+    <addNotNullConstraint columnName="is_nonconcurrent" tableName="qrtz_job_details"/>
+    <addColumn tableName="qrtz_job_details">
+      <column name="is_update_data" type="boolean"/>
+    </addColumn>
+    <sql>UPDATE qrtz_job_details SET is_update_data = is_stateful;</sql>
+    <addNotNullConstraint columnName="is_update_data" tableName="qrtz_job_details"/>
+    <dropColumn columnName="is_stateful" tableName="qrtz_job_details"/>
+
+    <addColumn tableName="qrtz_fired_triggers">
+      <column name="is_nonconcurrent" type="boolean"/>
+    </addColumn>
+    <sql>UPDATE qrtz_fired_triggers SET is_nonconcurrent = is_stateful;</sql>
+    <!--
+    <addNotNullConstraint columnName="is_nonconcurrent" tableName="qrtz_fired_triggers"/>
+    <addColumn tableName="qrtz_fired_triggers">
+      <column name="is_update_data" type="boolean"/>
+    </addColumn>
+    <sql>UPDATE qrtz_fired_triggers SET is_update_data = is_stateful;</sql>
+    <addNotNullConstraint columnName="is_update_data" tableName="qrtz_fired_triggers"/>
+    -->
+    <dropColumn columnName="is_stateful" tableName="qrtz_fired_triggers"/>
+
+
+    <!-- Add new 'sched_name' column to all tables -->
+
+    <addColumn tableName="qrtz_blob_triggers">
+      <column name="sched_name" type="varchar(120)" value="TestScheduler"/>
```

Comment:
This is a problem: we need to replace "TestScheduler" with our durable scheduler name, "databaseReport".